core.rb

```diff
@@ -632,6 +637,12 @@ def Core.repo_exists_in_github? (repos_url)
     end
   end
 
+  def Core.repo_caps_check? (repos_url, full_repos_url)
```

Comment:
To be sure I understand, we already check that the repository and the package have the same name / capitalization, and now we are making sure that the repository and URL have the same capitalization? Does it make better sense to use the DESCRIPTION file as definitive for the package name, i.e., doing whatever is done to parse the DESCRIPTION Package: field for the check between repository & package?

Reply:
The issue was in the sqlite database that has the github link to map the issue -  so this is done before we even get to checking or looking at the package

Reply:
so on issues.bioconductor.org in the sqlite database -   it gets filled -  and the name field needed to match the github repository capitalization
```
id|name|pw_hash|issue_number|login
1|dtenenba/spbtest|$2a$10$X1QM9XUSj94zxjtbXbtEpeQ4gAFY5NLCDLkgMjA6sOkkCtchJM5p6|4|dtenenba
```